### PR TITLE
Redirect filenames/slugs with parentheses to non-parentheses equivalents

### LINKED ISSUE #13936: Redirection of URL with parenthesis()
## What page(s) did you find the problem on?

https://developer.mozilla.org/en-US/docs/Web/CSS/color_value/color-contrast()

## What is the problem?

The url without the parenthesis at the end: [https://developer.mozilla.org/en-US/docs/Web/CSS/color_value/color-contrast](https://developer.mozilla.org/en-US/docs/Web/CSS/color_value/color-contrast) is not redirected to the version with the parenthesis.

This causes problems when [sharing the url](https://twitter.com/DctStrangelove/status/1504007889316364288) on Twitter and probably other plateforms that don't accept the parenthesis.

## What did you expect to see?

I expect the URL without parenthesis to be redirected to the actual URL, with parenthesis.

This is what happens with https://developer.mozilla.org/en-US/docs/Web/CSS/clamp
I think the behavior should be the same for all page ending with parenthesis (redirection or not)

Thanks for your work!

### LINKED ISSUE #14032: Content suggestion: <TITLE OF SUGGESTION>

### PR DESCRIPTION
**Pull request by sideshowbarker**
Fixes https://github.com/mdn/content/issues/13936. In the CSS substree, we have 76 articles whose filenames and slugs contain parentheses. Those parentheses create a number of problems — including that many or most URL auto-linkifiers don’t recognize the parentheses as being part of the URL, and so only linkify the part before the parentheses; one place where that happens is Twitter; see, e.g., https://twitter.com/DctStrangelove/status/1504007889316364288

And another problem for our own maintenance is that command-line shells don’t expect parenthesis characters in filenames — because those characters have special meanings to shells (and to regex engines, etc.), and so any time want to do something with those filenames or slugs for the command line, we need to quote the filenames or slugs.

### CODE REVIEW

File: files/en-us/web/css/abs/index.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 title: abs()
-slug: Web/CSS/abs()
+slug: web/css/abs
```
Comment (caugner) [inline files/en-us/web/css/abs/index.md:3]: Some of these have `web/css` written in lowercase. Is that one purpose?

Comment (sideshowbarker) [reply files/en-us/web/css/abs/index.md:3]: > Some of these have `web/css` written in lowercase. Is that one purpose?

No, that was an accidental artifact of the script I wrote to make the changes. I’m correcting it now and will force-push a fix soon.

**Review by teoli2003 — APPROVED** [review]
I'm fine with this (fit-content is the only unusual case now).

Will this create a lot of redirect flaws? If so should we, in a follow-up PR, update the links?

(I would like a second person approving, so I'm approving but not merging) [oops, it is a draft anyway, sorry!]

**Comment by sideshowbarker** [comment]
> Will this create a lot of redirect flaws?

It did, yes — but I’ve now pushed an additional commit to this PR branch, with a change that fixes every new redirect flaw introduced by the filename/slug renaming.

PR preview here still shows about 18 redirect flaws, but those are all existing flaws — not new ones introduced here — and anyway, those flaws are mostly in old parts of the `/en-US/docs/Mozilla/Firefox/Releases` tree.

> If so should we, in a follow-up PR, update the links?

I went ahead and updated them here — because here, the PR preview can show us exactly which flaws the renaming introduced, so that we can ensure the fixes get made here.

If we were to do it in a follow-up PR instead, it seems like we’d lose the ability to identify what fixes need to be made.

The downside is that this ends up being a much bigger patch: 202 files changed…

**Comment by sideshowbarker** [comment]
I think I finally managed to catch everything, so this should now be ready

**Review by teoli2003 — APPROVED** [review]
Looks good to me. I didn't check the whole last commit but my spot-checks were ok.

**Comment by teoli2003** [comment]
Let's merge this before it rots.

**Pull request merged**